### PR TITLE
Add markdown linter

### PR DESCRIPTION
**Pull request by hanhsuan**
Use markdownlint-cli2 to check Markdown content by github action, and fix errors that be found by markdownlint-cli2.

### CODE REVIEW

File: Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf
Comment (LiaoU3) [inline Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf:1]: Why is this PR change this again?

**Review by LiaoU3 — CHANGES_REQUESTED** [review]

File: Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf [hunk 1]
```diff
@@ -14,10 +14,6 @@ OPEN_BG_SSID = cert-bg-open-tel-l3-01
 OPEN_N_SSID = cert-n-open-tel-l3-01
 OPEN_AC_SSID = cert-ac-open-tel-l3-01
 OPEN_AX_SSID = cert-ax-open-tel-l3-01
-# === Select one BTDEVADDR below ===
-BTDEVADDR = 7C:B2:7D:4B:14:95
-# BTDEVADDR = 28:3A:4D:46:79:C0
-# BTDEVADDR = 34:6F:24:A8:93:EE
-# ==================================
+BTDEVADDR = 7C:B2:7D:4B:14:95,28:3A:4D:46:79:C0,34:6F:24:A8:93:EE,80:32:53:D8:0D:1E
```
Comment (LiaoU3) [inline Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf:17]: Why is this PR change this again?

**Review by LiaoU3 — CHANGES_REQUESTED** [review]

File: Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf (hunk 1 above)
Comment (hanhsuan) [reply Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf]: I have fixed that I checkout from non-main branch.

**Review by baconYao — APPROVED** [review]
LGTM

**Comment by stanley31huang** [comment]
@hanhsuan  we have highlight bash/shell commands in markdown files with different alias, likes sh, bash, shell.
will they all present one command style?

**Comment by hanhsuan** [comment]
@stanley31huang I've changed `shell/bash/sh` to the same alias `sh`

File: Tools/PC/jira-card-creator/README.md [hunk 2]
```diff
@@ -43,16 +49,17 @@ Here is a brief explanation about each part:
 
 #### 0. Setup virtual environment
 
-```bash
-$ source setup.sh
+```sh
+source setup.sh
```
Comment (stanley31huang) [inline Tools/PC/jira-card-creator/README.md:53]: @hanhsuan  I think the original purpose is tell user to execute this command in shell terminal. so could you please add the "$" character back?

Comment (hanhsuan) [reply Tools/PC/jira-card-creator/README.md:53]: @stanley31huang this is an [issue](https://github.com/DavidAnson/markdownlint/blob/5a4cf179c42646a0ad48c3f9522fe324849d73ba/doc/md014.md) found by linter. If this is must have, I could add comment to ignore it.

**Review by stanley31huang — APPROVED** [review]
this PR is ready to merge.

**Pull request merged**